### PR TITLE
[sc-165634] Remove outdated documentation about repoName use for monorepos

### PR DESCRIPTION
**Pull request by jazanne**
Since we introduced monorepo support, we no longer need to tell people to use unique repo name / project combo

Also fixes some lint errors in action.yaml

### CODE REVIEW

**Review by tiffanylphan — APPROVED** [review]

File: build/metadata/github-actions/action.yml [hunk 1]
```diff
@@ -9,30 +9,36 @@ inputs:
...
   ignoreServiceErrors:
     description: If enabled, the scanner will terminate with exit code 0 when the LaunchDarkly API is unreachable or returns an unexpected response.
-    default: false
+    default: 'false'
+    required: false
   lookback:
-    description: 'Sets the number of Git commits to search in history for
-whether a feature flag was removed from code. May be set to 0 to disabled this feature. Setting this option to a high value will increase search time.'
-    default: 10
+    description: Set the number of commits to search in history for whether a feature flag was removed from code. May be set to 0 to disabled this feature. Setting this option to a high value will increase search time.
```
Comment (ember-stevens) [inline build/metadata/github-actions/action.yml:39]:
```suggestion
    description: Set the number of commits to search in history for whether you removed a feature flag from code. You may set to 0 to disable this feature. Setting this option to a high value will increase search time.
```

**Review by ember-stevens — CHANGES_REQUESTED** [review]

File: docs/RELEASING.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
 
-## Github Releases
+## GitHub Releases
 
-This project uses [goreleaser](https://goreleaser.com/) to generate github releases. Releases are automated via CircleCI. To generate a new release, simply tag the commit you want to release and push the tag. If the tag ends in -rc(.+), the github release will be marked as "Pre-release." If you'd like to see how release notes are generated, see the .circleci/config.yml publish job.
+This project uses [goreleaser](https://goreleaser.com/) to generate GitHub releases. Releases are automated via CircleCI. To generate a new release, simply tag the commit you want to release and push the tag. If the tag ends in -rc(.+), the github release will be marked as "Pre-release." If you'd like to see how release notes are generated, see the .circleci/config.yml publish job.
```
Comment (mmrj) [inline docs/RELEASING.md:8]:
```suggestion
This project uses [goreleaser](https://goreleaser.com/) to generate GitHub releases. Releases are automated via CircleCI. To generate a new release, tag the commit you want to release and push the tag. If the tag ends in -rc(.+), the github release will be marked as "Pre-release." If you'd like to see how release notes are generated, see the .circleci/config.yml publish job.
```

File: docs/RELEASING.md [hunk 3]
```diff
@@ -23,4 +23,4 @@ Run `make publish-dev-circle-orb TAG=$VERSION` or `make-publish-release-circle-o
 
 ## Beta builds
 
-To push a beta build, set the `PRERELEASE=true` environment variable before running a release task. e.g. `make publish-all TAG=1.0.0-beta1`. Note: to publish a beta circle ci orb, run `make publish-dev-circle-orb TAG=$VERSION`
\ No newline at end of file
+To push a beta build, set the `PRERELEASE=true` environment variable before running a release task. e.g. `make publish-all TAG=1.0.0-beta1`. Note: to publish a beta circle ci orb, run `make publish-dev-circle-orb TAG=$VERSION`
```
Comment (mmrj) [inline docs/RELEASING.md:26]:
```suggestion
To push a beta build, set the `PRERELEASE=true` environment variable before running a release task, for example, `make publish-all TAG=1.0.0-beta1`. To publish a beta CircleCI orb, run `make publish-dev-circle-orb TAG=$VERSION`.
```

File: build/metadata/github-actions/action.yml [hunk 4]
```diff
@@ -9,30 +9,36 @@ inputs:
...
   contextLines:
     description:
       The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, no source code will be sent to LaunchDarkly. If 0, only the lines containing flag references will be sent. If > 0, will send that number of context lines above and below the flag reference. A maximum of 5 context lines may be provided.
-    default: 2
+    default: '2'
+    required: false
   allowTags:
-    description: Enables storing references for tags. The tag will be listed as a branch.
-    default: false
+    description: Enable storing references for tags. The tag will be listed as a branch.
```
Comment (mmrj) [inline build/metadata/github-actions/action.yml:27]: Okay that lines 27, 31, 39 don't have single-quotes around the `description` content, but lines 12, 15, 18 do? I'm not familiar with the action format and not sure how to test.

**Review by mmrj — CHANGES_REQUESTED** [review]

File: build/metadata/github-actions/action.yml (hunk 4 above)
Comment (jazanne) [reply build/metadata/github-actions/action.yml:27]: i believe they only _need_ to be wrapped in quotes if they contain a subset of special characters or spaces? But i can wrap all in quotes for consistency

Comment (mmrj) [reply build/metadata/github-actions/action.yml:26]: Up to you if it's not required, I just wasn't sure. Thanks for considering!

**Review by mmrj — APPROVED** [review]

**Pull request merged**